### PR TITLE
Change default hdu to 0 for (assumed) image FITS hdulists

### PR DESCRIPTION
**Pull request by keflavich**
See #118

### CODE REVIEW

File: reproject/healpix/high_level.py [hunk 1]
```diff
@@ -5,7 +5,9 @@
 __all__ = ['reproject_from_healpix', 'reproject_to_healpix']
 
 
-def reproject_from_healpix(input_data, output_projection, shape_out=None, hdu_in=None, order='bilinear', nested=False, field=0):
+def reproject_from_healpix(input_data, output_projection, shape_out=None,
+                           hdu_in=None, order='bilinear', nested=False,
```
Comment (keflavich) [inline reproject/healpix/high_level.py]: Yes.

File: reproject/healpix/utils.py [hunk 2]
```diff
@@ -49,7 +49,8 @@ def parse_input_healpix_data(input_data, field=0, hdu_in=None):
 
     if isinstance(input_data, six.string_types):
         from healpy import read_map
-        array_in, header = read_map(input_data, verbose=False, h=True, field=field, hdu=1 if hdu_in is None else hdu_in)
+        array_in, header = read_map(input_data, verbose=False, h=True, field=field,
+                                    hdu=1 if hdu_in is None else hdu_in)
```
Comment (astrofrog) [inline reproject/healpix/utils.py]: Would it be worth just defaulting to 1 in this function too and avoiding the 'if' logic here?

**Comment by astrofrog** [comment]
@keflavich - can you rebase? Thanks!

**Comment by keflavich** [comment]
done.  I think this is a case where a merge would have been easier than a rebase; I feel reasonably confident I got things wrong.

File: reproject/healpix/high_level.py [hunk 3]
```diff
@@ -5,7 +5,9 @@
 __all__ = ['reproject_from_healpix', 'reproject_to_healpix']
 
 
-def reproject_from_healpix(input_data, output_projection, shape_out=None, hdu_in=None, order='bilinear', nested=None, field=0):
+def reproject_from_healpix(input_data, output_projection, shape_out=None,
+                           hdu_in=None, order='bilinear', nested=None,
```
Comment (astrofrog) [inline reproject/healpix/high_level.py]: Should this be changed to 1?

Comment (keflavich) [reply reproject/healpix/high_level.py]: `hdu_in=1`?  Maybe.  I got confused which was old and which was new...

Comment (astrofrog) [reply reproject/healpix/high_level.py]: Yes, ``hdu_in`` should be 1 I think

**Comment by astrofrog** [comment]
Thanks!

**Pull request merged**